docs/content/reference/cli/authelia/authelia_storage_bans_ip.md

```
@@ -0,0 +1,69 @@
+
+Manages ip bans
+
+### Synopsis
+
+Manages ip bans.
+
+This subcommand allows listing, creating, and revoking ip bans from the regulation system.
+
+### Examples
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)